rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
Make sure these two instance variables will also be `attr_accessor` since we want to `get` and `set` for both.